fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -3015,7 +3016,7 @@ public void refreshMaterializedView(String dbName, String mvName, int priority)
             Task task = TaskBuilder.buildMvTask(materializedView, dbName);
             taskManager.createTask(task, false);
         }
-        taskManager.executeTask(mvTaskName, priority);
+        taskManager.executeTask(mvTaskName, new ExecuteOption(priority));
```

Comment:
it seems that mergeRedundant is default false. It means you disable merge by default?

Reply:
Yes. For the periodical tasks generated by our system, I think merge can be turned on, and mrege should not be done by default for user trigger.